### ISSUE TITLE
Binary distribution

### ISSUE THREAD

**Issue by sshine** [NorfairKing/smos#236]
I managed to install smos on a Debian 11 VPS with 2GB RAM via Stack, but I felt that it required non-trivial steps: Using the system Stack would not give very clear errors that certain C libraries were missing in order to successfully install GHC 8.10.7, and what the corresponding Debian packages were called. Reinstalling Stack via GHCup gave some hints about this that were easy to follow. Smos itself requires zlib beyond that. Then there was the issue of having to install a swap file on the VPS because compilation of Haskell takes up a lot of memory. Installing via nix might have resulted in fewer error states, but I'm not shopping for two operating systems in one day.

In summary: It was possible to install smos on a Debian 11, but the barrier of entry seemed a little high for someone who already knew the toolchain. I wonder if a binary distribution would make smos more widely accessible.

**Comment by NorfairKing** [comment]
I would love to, but good luck.
I've tried to build statically linked binaries in the past but it's entirely non-trivial.

Realistically, the options are currently:

1. Figure it out using the docs
2. Use the nix build
3. Use the nixos integration
4. Use the web browser

**Comment by NorfairKing** [comment]
A statically linked binary would be great but I can't commit to supporting it.
NixOS with home-manager is currently the most supported option.
If someone comes by and finds a way to have CI maintain statically linked binaries, they're welcome to add that.
